ui/public/config.json

```diff
@@ -96,5 +96,6 @@
   "basicZoneEnabled": true,
   "multipleServer": false,
   "allowSettingTheme": true,
+  "forgotPasswordEnabled": false,
```

Comment:
Please try if this can be removed by making any usual checks of API availability.